orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can I just tip my hat to you for the use of hyperlinks? Brilliant!</p><p> </p><p>Spell check might be needed ;) see point 1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thank you! Though my concern is that once it goes on the live website the hyperlinks become inactive. They become static text. Haven't found a way around this yet.</p>

Reply:
Do the following two steps. It will work (but you won't be able to see it in your ipynb, sadly):
### Step 1
Change the opening part to the following:
```
In this lecture we aim to cover the following topics:
1. [](content:orbit-maneuvers)
2. [](content:hohmann-transfer)
3. [](content:bi-elliptic-transfer)
4. [](content:non-tangential-transfer)
5. [](content:plane-change-maneuver)
```

### Step 2
Change each of the section headings as per below:

```
(content:orbit-maneuvers)=
## Orbit Maneuvers
Content about orbit maneuvers.

(content:hohmann-transfer)=
## Hohmann Transfer
Content about Hohmann transfer.

(content:bi-elliptic-transfer)=
## Bi-elliptic transfer
Content about bi-elliptic transfer.

(content:non-tangential-transfer)=
## Non tangential transfer
Content about non tangential transfer.

(content:plane-change-maneuver)=
## Plane Change maneuver
Content about plane change maneuver.
```

Reply:
@mitanshc @Emma-airi Please note that that I have added two new files to master. So you will have to:
1. `checkout master`
2. `fetch` from the `upstream` remote
3. `merge` from the `upstream` remote
4. This is a new step: you can update your `main` branch on Github (aka `origin`) with a `push`.
5.  Then, checkout the respective branches you are working on and `merge main`.
6. Then do whatever work you need to and follow the old `add`, `commit`, and `push` sequence.

The specific commands you need to run are also described [here](https://github.com/angadhn/SpacecraftDynamics/blob/main/CONTRIBUTING.md#keeping-your-fork-updated) in the new file with instructions on how to contribute.

Do let me know if you have any questions or confusion.

Reply:
<p>The above didn't work for the hyperlinks, but I managed to figure out with some ChatGPT-ing:</p><ul><li>Create an anchor at the site we want to go to on the page using</li></ul>

<pre class="ql-syntax" spellcheck="false">&lt;a name = "orbit-maneuvers"&gt;&lt;/a&gt;
## Orbit Maneuvers
</pre><p> </p><ul><li>Then go to the contents section at the top and then add:</li></ul>

<pre class="ql-syntax" spellcheck="false">[Orbit Maneuvers](#orbit-maneuvers)

</pre><p>It won't work in the app. reviewnb. It just opens a random new page. But When I converted to .html it works perfect. </p>

Reply:
I tried what you did; my suggestion is to just copy paste what I did in the Step 1 and Step 2.

Reply:
The reason of this, if you are curious, is that ChatGPT doesn't know that Jupyter Book is slightly different in how it works. This is described [here](https://jupyterbook.org/en/stable/content/references.html#reference-section-labels); there are two options and I have decided that we will go with the second option as that is closer to what you started with. It also looks more close to normal markdown. Hope this helps.

Reply:
No, you don't have to do that. Just copy paste my instructions.